sessionctx/context.go

```diff
@@ -34,7 +34,8 @@ type Context interface {
 	NewTxn() error
 
 	// Txn returns the current transaction which is created before executing a statement.
-	Txn() kv.Transaction
+	// The returned kv.Transaction is not nil, but maybe pending or invalid.
+	Txn(...struct{}) kv.Transaction
```

Comment:
If we use a bool parameter, how many lines need to be modified?

Reply:
A lot, you can grep `Txn()` in the code base.
I don't want that to be messed with the changes in this PR